### ISSUE TITLE
Flow result is  a mess when the adjcent two frames are almost the same(no action or very small action).

### ISSUE THREAD

**Issue by jucic** [princeton-vl/RAFT#123]
![image](https://user-images.githubusercontent.com/17615552/142862210-15d98a9f-9c34-4b7c-baea-71bfe12b5e90.png)
look at the above picture, when the adjcent two frames are almost the same(no action or very small action), Flow result is  a mess, I guess this is because there is no static scene in the training dataset of RAFT(Flyingchairs,Flyingthings,sintel and so no), so I made some dataset of static scene(optical flow always equal to 0 ) and finetuned RAFT after sintel stage of train_standard.sh，however, it doesn't imporve sufficiently(still comes to a mess in this condition sometimes), anyone knows why and  how to solve this? Thanks in advance!

**Comment by eyildiz-ugoe** [comment]
It's exactly the same story for me. I have a scene in which only one object rotates slowly and I get these messy frames often. 

One solution to this would be to train the network from the scratch using our own data. Does anyone have instructions for that?

**Comment by UcefMountacer** [comment]
yeah, same for my situation. The model output is a mess it at some frames. The situation I'm doing it for involves a small object rotating, and a constant scene.

Here is a comparison between a good output and a messy one:
![image](https://user-images.githubusercontent.com/47783157/147247103-80e75e53-fbb3-405d-8e2a-740d8ce379a3.png)
![image](https://user-images.githubusercontent.com/47783157/147247156-e25a6db8-2c0d-4efa-a509-af122ceabc57.png)

**Comment by ckcraig01** [comment]
I have also observed this condition, any ideas?

**Comment by ckcraig01** [comment]
Dear all, this is because of the normalization issue, this flow_norm can solve all your problem:
https://github.com/tomrunia/OpticalFlow_Visualization/pull/7

**Comment by jucic** [comment]
> Dear all, this is because of the normalization issue, this flow_norm can solve all your problem: [tomrunia/OpticalFlow_Visualization#7](https://github.com/tomrunia/OpticalFlow_Visualization/pull/7)

oh my god! Thanks a lot! I can't beleive this simple problem should have confused me so long time

**Comment by hirominn** [comment]
This is where the normalization takes place. I think you can display with fixed saturation by editing here.  
https://github.com/princeton-vl/RAFT/blob/aac9dd54726caf2cf81d8661b07663e220c5586d/core/utils/flow_viz.py#L125-L131
